sensing/autoware_pointcloud_preprocessor/launch/polygon_remover.launch.py

```diff
@@ -34,7 +34,7 @@ def generate_launch_description():
 
     my_component = ComposableNode(
         package=pkg,
```

Comment:
the variable pkg does not have the prefix it seems

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/7983/commits/77d9a06386901af0fe254d792fc1af3f87fdec27